src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -508,6 +522,34 @@ bool WholeBodyDynamicsDevice::openExternalWrenchesPorts(os::Searchable& config)
     return ok;
 }
 
+bool WholeBodyDynamicsDevice::openFilteredFTPorts(os::Searchable& config)
+{
+    bool ok = true;
+        // create port names
+
+        std::string sensorName;
+        std::string portName;
+        outputFTPorts.resize(estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE));
+        for(int ft=0; ft < estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE); ft++ )
+        {
+            sensorName= estimator.sensors().getSensor(iDynTree::SIX_AXIS_FORCE_TORQUE,ft)->getName() ;
+
+            yDebug() << "wholeBodyDynamics: creating port names for  " << sensorName;
```

Comment:
I think we can either remove this messages (my preferred solution) or transform it in `yInfo`.

Reply:
Change it to info, its kind of useful for me when launching to verify a couple of steps. If after the tests with the temperature I see is not so useful anymore I will remove it.